packages/venia-ui/lib/components/CartPage/ProductListing/EditModal/editModal.js

```diff
@@ -11,16 +11,20 @@ import ProductForm from './productForm';
 
 const EditModal = props => {
     const { item, setIsCartUpdating } = props;
-    const talonProps = useEditModal();
-    const { handleClose, isOpen } = talonProps;
+    const talonProps = useEditModal({ item });
```

Comment:
I notice you pass the item here - is that to do the calculation of the variant price on initial render? You aren't using it anymore in the talon though.

Reply:
I attempted your suggestion to initialize in the talon at first, but went the simple route. Will remove this prop.